packages/api-specification/src/new-window-api.html

```diff
@@ -38,7 +38,7 @@
 <h3>Method Signature</h3>
 
 <pre>
-window.open(url, name, features)
+ssf.Window(url, name, features)
```

Comment:
Yeah sounds like a good idea. Won't be a big change if the feedback is to use the constructor pattern :)